streams/src/main/java/org/apache/kafka/streams/kstream/ValueMapper.java

```diff
@@ -18,12 +18,18 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * The KeyValueMapper interface for mapping an original value to a new value (could be another key-value pair).
+ * The {@link KeyValueMapper} interface for mapping an original value to a new value (could be another key-value pair).
```

Comment:
What about:

> Defines a stateless transformation of a record value into a new value.

(Not sure whether this wording approach is consistent with other interface docstrings.)